tools/packaging/kernel/build-kernel.sh

```diff
@@ -110,7 +110,7 @@ Options:
 	-t <hypervisor>	: Hypervisor_target.
 	-u <url>	: Kernel URL to be used to download the kernel tarball.
 	-v <version>	: Kernel version to use if kernel path not provided.
-	-x <type>	: Confidential guest protection type, such as sev, snp and tdx
+	-x <type>	: Confidential guest protection type, such as sev, snp, tdx. or "confidential" (for all of those).
```

Comment:
minor nit: switch full-stop for comma (or just remove entirely if you aren't a fan of the Oxford comma)? I'm also very slightly triggered by the change in alignment for url and type, but that wasn't added in this PR, so I can get over it easily!
```suggestion
	-x <type>	: Confidential guest protection type, such as sev, snp, tdx, or "confidential" (for all of those).
```